routing/graph.go

```diff
@@ -86,3 +92,38 @@ func (g *CachedGraph) fetchNodeFeatures(nodePub route.Vertex) (
 
 	return g.graph.FetchNodeFeatures(nodePub)
 }
+
+// FetchAmountPairCapacity determines the maximal capacity between two nodes.
+//
+// NOTE: Part of the routingGraph interface.
+func (g *CachedGraph) FetchAmountPairCapacity(nodeFrom, nodeTo route.Vertex,
+	amount lnwire.MilliSatoshi) (btcutil.Amount, error) {
+
+	// For the local node we assume no information on the channel capacity.
+	if nodeFrom == g.sourceNode() {
+		return 0, nil
+	}
+
+	// Create unified policies for all incoming connections.
+	u := newUnifiedPolicies(g.source, nodeTo, nil)
+
+	err := u.addGraphPolicies(g)
+	if err != nil {
+		return 0, err
+	}
+
+	// We may not have all policies available to describe the hop between
+	// the nodes (in the case of hop hints), which is why we return 0 in
+	// this case.
+	unifiedPolicy, ok := u.policies[nodeFrom]
+	if !ok {
+		return 0, nil
+	}
+
+	policy := unifiedPolicy.getPolicy(amount, nil)
```

Comment:
Add comment why no bandwidth hints here? Or can you call `getPolicyNetwork` directly now that it has been ruled out that we are routing from the source node?